src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -84,7 +84,12 @@ override void OnLaunched(LaunchActivatedEventArgs e)
 #endif
 			var sw = Stopwatch.StartNew();
 			var n = Windows.UI.Xaml.Window.Current.Dispatcher.RunIdleAsync(
-				_ => Console.WriteLine("Done loading " + sw.Elapsed));
+				_ =>
+				{
+					Console.WriteLine("Done loading " + sw.Elapsed);
+					Console.WriteLine($"DependencyPropertyDetailsCollection: reused: {DependencyPropertyDetailsCollection._pool.Reused} created: {DependencyPropertyDetailsCollection._pool.Created}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5420501)